src/store/modules/nft.js

```diff
@@ -307,31 +305,31 @@ const actions = {
     classIdSet.add(classId);
     commit(TYPES.NFT_SET_USER_NFT_CLASS_FEATURED_SET_MAP, {
       address,
-      classIdSet,
+      classIdSet: new Set(classIdSet), // clone to trigger reactivity
```

Comment:
How about using `Vue.set` in the `NFT_SET_USER_NFT_CLASS_FEATURED_SET_MAP` mutation?

Reply:
Already used `Vue.set()` in [mutation](https://github.com/likecoin/liker-land/pull/791/files#diff-bc31414c5136bf8f64828fe6722217f2a2dc6a0e66fc238cce705547403f6dc2R38-R49). Seems VueX only triggers UI reactivity when the pointer in the field of object state (`state.userNFTClassFeaturedSetMap`) changed.